### PR TITLE
:sparkles: Show commit hash, build time and more with version info

### PR DESCRIPTION
**Pull request by rinx**
Signed-off-by: Rintaro Okamura <rintaro.okamura@gmail.com>

### Description:

I added a function that shows version, commit hash, build time and extra info.
Extra info includes Go version, OS, architecture, cgoflag, and NGT version.

Example:
```
% docker run -it vdaas/vald-agent-ngt:latest --version
2019-12-25 02:59:13     [INFO]: vald agent ngt server
2019-12-25 02:59:13     [INFO]: version     -> v0.0.1
2019-12-25 02:59:13     [INFO]: commit hash -> 7da4736ff2006a7d9cbb2320d2e9480e129a5bd7
2019-12-25 02:59:13     [INFO]: build time  -> Wed, 25 Dec 2019 02:59:13 UTC
2019-12-25 02:59:13     [INFO]: arch        -> amd64
2019-12-25 02:59:13     [INFO]: cgo enabled -> 1
2019-12-25 02:59:13     [INFO]: go version  -> 1.13.5
2019-12-25 02:59:13     [INFO]: ngt version -> 1.8.4
2019-12-25 02:59:13     [INFO]: os          -> linux
```

### Related Issue:

Nothing.

### How Has This Been Tested?:

Docker images are built in my local machine and tested with `--version` flag.

### Environment:

- Golang Version: 1.13.5
- Docker Version: 19.03.5
- Kubernetes Version: 1.16.3
- NGT Version: 1.8.4

### Types of changes:

- [ ] Bug fix [type/bug]
- [X] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensure all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: internal/info/info.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+				maxlen = l
+			}
+		}
+
+		infoFormat := fmt.Sprintf("%%-%ds -> %%s", maxlen)
+
+		log.Infof("vald %s server", name)
+		log.Infof(infoFormat, defaultKeys[0], log.Bold(Version))
+		log.Infof(infoFormat, defaultKeys[1], GitCommit)
+		log.Infof(infoFormat, defaultKeys[2], BuildTime)
```
Comment (kpango) [inline internal/info/info.go]: could you please build all information as string variable then execute log.Info once

Comment (rinx) [reply internal/info/info.go]: okay. i revised it in 3ba0575 .
it shows server information as below. Is it okay?

```
% docker run -it vdaas/vald-agent-ngt:latest --version
2019-12-28 09:12:26     [INFO]: vald agent ngt server
version     -> v0.0.1
commit hash -> c347ee8f7c1b148019ffb3f22f106ff4622f6c05
build time  -> Sat, 28 Dec 2019 09:12:26 UTC
arch        -> amd64
cgo enabled -> 1
go version  -> 1.13.5
ngt version -> 1.8.4
os          -> linux
```

**Comment by rinx** [comment]
aha, i see.

**Comment by rinx** [comment]
Now it's like this:
```
% docker run -it vdaas/vald-agent-ngt:latest --version
2019-12-28 09:27:51     [INFO]:
vald agent ngt server
version     -> v0.0.1
commit hash -> 3ba0575852dee5c76bc43f076b6a926617b6f5d2
build time  -> Sat, 28 Dec 2019 09:27:51 UTC
arch        -> amd64
cgo enabled -> 1
go version  -> 1.13.5
ngt version -> 1.8.4
os          -> linux
```

**Comment by kpango** [comment]
LGTM

**Pull request merged**